image_test/configuration/linux/configuration-test.py

```diff
@@ -20,7 +20,7 @@
 
 def main():
   # detect distros to instantiate the correct class
-  if sys.version_info >= (3, 6):
+  if sys.version_info >= (3, 7):
```

Comment:
I think this was accurate since 3.6 and above have distro.

Reply:
In FreeBSD, using python 3.6, we do not have the module `distro` by default and it raises an error. And since [`platform` was removed only on python 3.8](https://bugs.python.org/issue28167) I only increased this version constraint. If you do not agree with that approach, we can find a way to install  `pip` and then `distro` in FreeBSD via `bootstrap.sh`.

Reply:
SGTM